accounts/views.py

```diff
@@ -202,6 +209,16 @@ def complete_profile(request):
 
     return render(request, 'accounts/volunteer_profile.html')
 
+def ajax_volunteer_rollcheck(request):
+    rollNo = request.POST['roll']
+    roll_no = Volunteer.objects.filter(roll_no=rollNo)
+    data = {}
+    if roll_no.exists():
+        data['isExist'] = True
+        return JsonResponse(data)
+    else:
+        data['isExist'] = False
+        return JsonResponse(data)
```

Comment:
Put return statement out of the if-else block as both the return statement are infact the same. Also, you may use the ternary operator instead of if-else which will reduce the code to a single line.